### DISCUSSION TITLE
Request option to choose codec when add tv series

### DISCUSSION THREAD

**Discussion by lordzerg** [pymedusa/Medusa#10803] [Ideas & Feature requests]
Hello, my question or requests is to be able to choose the codec when adding new tv series or maybe later with Edit.
The option can be for video codec HEVC/H.265/x265 and x264 and maybe for sound codec or more important would be the channels: 2 ch. or 5+1ch.
If this can be implemented would be great will be easier to set my preference :)

**Comment by p0psicles** [comment]
You can do this using `required words`

**Reply by lordzerg** [reply]
My idea is if this can be added to the list with resolutions (Allowed and Preferred quality, where we choose 720/1080p web-dl etc...) I am currently using the same method 'required words' as you said.

**Reply by p0psicles** [reply]
Unfortunately, because of technical limitations that's not possible. In an ideal situation you'd be able to mix and match resolution, container, encoding, etc. And create your own quality "profiles". That's something for the future. In the mean time, we'd have to do with required words.
